week3/homework/src/todos.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
What is a json file with data doing in a `src` directory?

Reply:
I forgot to delete json file after I test post requests.